.happy/terraform/modules/schema_migration/main.tf

```
@@ -44,15 +44,15 @@ resource aws_batch_job_definition schema_migrations_swap {
     resourceRequirements = [
       {
         type= "VCPU",
-        Value="6"
+        Value="38"
```

don't need to leave a comment, but curious for myself why these numbers? educated estimates based on testing? some calculation?

---

the 38 is MEMORY/8GB. All of AWS machines in `r5a` family follow this pattern. I don't have a strong reason for following this, but if are going to use a machine to the point where nothing else can run than we mine as well use the whole thing.

```suggestion
        Value="32"
```